scripts/analytics/addon_snapshot.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from modularodm import Q
+
+from website.settings import ADDONS_AVAILABLE
+from website.app import init_app
+from website.settings import KEEN as keen_settings
+from keen.client import KeenClient
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_events():
+    counts = []
+    addons_available = {k: v for k, v in [(addon.short_name, addon) for addon in ADDONS_AVAILABLE]}
+    for short_name, addon in addons_available.iteritems():
+        user_count = addon.settings_models['user'].find().count() if addon.settings_models.get('user') else 0
+        node_count = addon.settings_models['node'].find().count() if addon.settings_models.get('node') else 0
+        deleted_count = addon.settings_models['node'].find(Q('deleted', 'eq', True)).count() if addon.settings_models.get('node') else 0
+        disconnected_count = addon.settings_models['node'].find(Q('external_account', 'eq', None) & Q('deleted', 'ne', True)).count() if addon.settings_models.get('node') else 0
+
+        counts.append({
+            'provider': {
+                'name': short_name
+            },
+            'users': {
+                'total': user_count
+            },
+            'nodes': {
+                'total': node_count,
+                'connected': node_count,
```

Comment:
Why are `total` and `connected` the same?  I would think `total` would be `connected + deleted + disconnected`.